test/rulesets/LinearAlgebra/matfun.jl

```diff
@@ -48,6 +53,11 @@
             A = Float64[0 10 0 0; -1 0 0 0; 0 0 0 0; -2 0 0 0]
             test_rrule(exp, A; check_inferred=false)
         end
+        @testset "imbalanced A with no squaring" begin
+            # https://github.com/JuliaDiff/ChainRules.jl/issues/595
+            A = randn(MersenneTwister(8), 3, 3)
```

Comment:
Maybe using a literal rather than risk the MersenneTwister changing?
(Like it has done in the past)
```suggestion
            A = [
                -0.007623430669065629 -0.567237096385192  0.4419041897734335;
                 2.090838913114862    -1.254084243281689 -0.04145771190198238;
                 2.3397892123412833   -0.6650489083959324 0.6387266010923911
                ]
```